src/main/java/org/sagebionetworks/web/client/GlobalApplicationStateImpl.java

```diff
@@ -190,17 +190,19 @@ public PlaceChanger getPlaceChanger() {
         new PlaceChanger() {
           @Override
           public void goTo(Place place) {
-            synapseJSNIUtils.setIsInnerProgrammaticHistoryChange();
-            // If we are not already on this page, go there.
-            if (!placeController.getWhere().equals(place)) {
-              try {
-                placeController.goTo(place);
-              } catch (Exception e) {
-                synapseJSNIUtils.consoleError(e.getMessage());
+            if (place != null) {
```

Comment:
Not really necessary after the bug fixes, but there were cases where the place token didn't resolve to a Place, so it was passed in as null.  This resulted in an uncaught js exception, so I check for null here.